### ISSUE TITLE
How  Access-Control-Allow-Origin *

### ISSUE THREAD

**Issue by jmikepaz** [jeske/SimpleHttpServer#14]
how to avoid corsets when requests from other hosts come in

**Comment by jeske** [comment]
Corsets?

**Comment by jmikepaz** [comment]
Yes, that's right, a request comes in from another host, the request is blocked by cors, how could I allow requests by enabling cors?
